docs/examples/eventrouter.yaml

```diff
@@ -113,30 +147,34 @@ spec:
                           image: gcr.io/heptio-images/eventrouter:v0.2
                           imagePullPolicy: IfNotPresent
                           volumeMounts:
-                          - name: config-volume
-                            mountPath: /etc/eventrouter
+                            - name: config-volume
+                              mountPath: /etc/eventrouter
                       serviceAccount: event-router-sa
                       volumes:
                         - name: config-volume
                           configMap:
                             name: event-router-cm
-
     delete:
       template: |
         apiVersion: argoproj.io/v1alpha1
         kind: Workflow
+        metadata:
+          name: er-delete-
         spec:
           entrypoint: delete-wf
           serviceAccountName: addon-manager-workflow-installer-sa
 
           templates:
             - name: delete-wf
+              retryStrategy:
+                limit: 2
+                retryPolicy: "Always"
               steps:
                 - - name: delete-ns
                     template: delete-ns
 
             - name: delete-ns
               container:
-                image: expert360/kubectl-awscli:v1.11.2
+                image: docker.intuit.com/dev/patterns/kubernetes/dev/kubectl-awscli:v1.19.11
```

Comment:
We shouldn't be using this and moreover this won't work

Reply:
This works though don't know the reason of not using this image. But, addon-manager mainly use basic commandline. previous version also works.
77929b3847f56f5dd2ba01525f98dd9a6aabc5e7

Reply:
Keiko is open source and we should be exposing internal artifacts to the public.

Reply:
A valid point.

Reply:
Looks like `expert360/kubectl-awscli:v1.18.6` is the latest available on Docker Hub.

Reply:
Yeah, and it satisfy our needs.

Reply:
Yes, used and tested it.